packages/grid/_modules_/grid/hooks/features/selection/gridSelectionSelector.ts

```diff
@@ -11,3 +13,15 @@ export const selectedGridRowsCountSelector: OutputSelector<
   gridSelectionStateSelector,
   (selection) => Object.keys(selection).length,
 );
+
+export const selectedGridRowsSelector = createSelector<
```

Comment:
I wonder. Are we all OK with turning state selectors into state transformation functions? This doesn't seem to be a light change. There is now an indirection.